### DISCUSSION TITLE
Does anyone know the pickit code for - enemy fire res? I want to find be able to find a perfect ravenlore

### DISCUSSION THREAD

**Discussion by danmm55** [blizzhackers/pickits#8] [Q&A about pickit codes]
What is the code for minus fire res if there is such a thing? the roll is kinda based on that stat. right now im set to pick up any ravenlores.

**Comment by mf022** [comment]
you can find ravenlore in - <https://github.com/blizzhackers/pickits/blob/master/d2nt/master/unique.nip#L179>, but also in some other pickit files

so **[passivefirepierce] >= ...**

**Comment by danmm55** [comment]
thanks thats what i needed. what about replenish life? some items roll this as a stat so itd be very helpful to know. i tried [itemreplenishlife] and about every other thing i could think of no no avail.

**Comment by mf022** [comment]
> what about replenish life?

in Autostat.js I found something but only for amulets <https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/common/AutoStat.js#L147> or rings <https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/common/AutoStat.js#L159>

Edit
like @e-n-g said  <https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/NTItemAlias.dbl#L885>

**Comment by e-n-g** [comment]
> what about replenish life?

[hpregen]
